docs/organizer-guide/quorum.rst

```diff
@@ -0,0 +1,159 @@
+# Write the Docs Meetup Quorum
```

Comment:
This just means it's not in the table of contents. I can add it if that's easier. Otherwise you can put it on this line: https://github.com/writethedocs/www/blame/main/docs/organizer-guide/index.rst#L26

Reply:
yes please and thank you for offering @ericholscher  I accept! Please add to TOC and also https://github.com/writethedocs/www/pull/1746 and any other wayward lame PRs that I created on writing day